openlibrary/core/yearly_reading_goals.py

```diff
@@ -52,6 +68,22 @@ def has_reached_goal(cls, username: str, year: int) -> bool:
         else:
             return results[0]['current'] >= results[0]['target']
 
+    @classmethod
+    def total_yearly_reading_goals(cls, since: date | None = None) -> int:
+        """Returns a Storage object of <Storage {'count': int}> where 'count' specifies the
+        number reading goals updated. `since` may be used
```

Comment:
```suggestion
    def total_yearly_reading_goals(cls, since: date | None = None) -> int:
        """Returns the number reading goals that were set. `since` may be used
```
Type hints and docstring are conflicting.  Return an `int` here.